sormas-api/src/main/resources/validations.properties

```diff
@@ -34,6 +34,9 @@ importEntryDoesNotExist = Invalid value %s in column %s; Entry does not exist in
 importEntryDoesNotExistDbOrCommunity = Invalid value %s in column %s; Entry does not exist in the database or in the specified community or is archived (Or maybe a wrong facility type was specified)
 importEntryDoesNotExistDbOrDistrict = Invalid value %s in column %s; Entry does not exist in the database or in the specified district or is archived (Or maybe a wrong facility type was specified)
 importEntryDoesNotExistDbOrRegion = Invalid value %s in column %s; Entry does not exist in the database or in the specified region or is archived
+importEntryRegionNotInJurisdiction = Invalid value %s in column %s; Specified region is not within you're user's jurisdiction
```

Comment:
"you're user's jurisdiction" -> "your jurisdiction" (for all three strings)

Also, rename "importEntryRegionNotInJurisdiction" to "importEntryRegionNotInUsersJurisdiction" to differentiate it from the other "notInJurisdiction" messages that are used to specify that the district of the imported entity does not belong to the region of the imported entity